Specifications/Language/2_Statements/README.md

```diff
@@ -1,42 +1,20 @@
 # Statements
 
-Q# distinguishes between statements and expressions. Q# programs consist of a mixture of classical and quantum computations, and the implementation looks much like any other classical programming language. Some statements, such as the `let` and `mutable` bindings, are well-known from classical languages, while others, such as conjugations or qubit allocations, are unique to the quantum domain.
+Q# distinguishes between statements and expressions. Q# programs consist of a mixture of classical and quantum computations, and the implementation looks much like any other classical programming language. Some statements, such as the `let` and `mutable` bindings, are well-known from classical languages, while others, such qubit allocations, are unique to the quantum domain.
 
 The following statements are currently available in Q#:
 
-* **Call statement**    
-    A call statement consists of an operation or function call returning `Unit`. The invoked callable needs to satisfy the requirements imposed by the current context. See [Call statements](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/2_Statements/CallStatements.md#call-statements) for more details.
-
-* **Return statement**    
-    A return statement terminates the execution within the current callable context and returns control to the caller. Any finalizing tasks are run after the return value is evaluated but before control is returned. See [Returns and termination](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/2_Statements/ReturnsAndTermination.md#returns-and-termination) for more details.
-
-* **Fail statement**    
-    A fail statement stops the entire program and collects information about the current program state before terminating in an error. It aggregates the collected information and presents it to the user along with the message specified as part of the statement. See [returns and termination](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/2_Statements/ReturnsAndTermination.md#returns-and-termination) for more details.
+* **Expression statement**    
+    Contains a Q# expression to be run, such as a call to an operation. If the last statement in a block is an expression statement, it may have its trailing semicolon omitted to give the block the evaluated value of the contained expression.
 
 * **Variable declaration**    
     Defines one or more local variables that are valid for the remainder of the current scope, and binds them to the specified values. Variables can be permanently bound or declared to be reassignable later on. See [Variable declarations and reassignments](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/2_Statements/VariableDeclarationsAndReassignments.md#variable-declarations-and-reassignments) for more details.
 
 * **Variable reassignment**    
```

Comment:
Technically, variable reassignments are an expression, not a statement.

Reply:
Oh I thought they were both statements. That's unfortunate because the docs talk about them together, and it is going to be difficult to go through and disentangle them. For now, maybe I can just drop this paragraph?

Reply:
Yeah, I think leaving them together makes sense, since splitting up declaring mutable variables and updating them makes the docs harder to navigate. I think dropping this paragraph is fine.

Reply:
I've dropped this paragraph here to minimize confusion, but I think a larger rewrite is going to be necessary to get this correct.